lib/features/composer/domain/usecases/download_image_as_base64_interactor.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:core/core.dart';
+import 'package:dartz/dartz.dart';
+import 'package:tmail_ui_user/features/composer/domain/repository/composer_repository.dart';
+import 'package:tmail_ui_user/features/composer/domain/state/download_image_as_base64_state.dart';
+
+class DownloadImageAsBase64Interactor {
+  final ComposerRepository _composerRepository;
+
+  DownloadImageAsBase64Interactor(this._composerRepository);
+
+  Future<Either<Failure, Success>> execute(String url) async {
```

Comment:
I think you should you Stream, first item is Loading state. Maybe we should showing loading in the position of inline image when downloading? WDYT?

Reply:
Ok. I think that's quite reasonable.